src/Tribe/Status/Abstract_Commerce.php

```diff
@@ -267,15 +257,66 @@ public function get_total_sale_tooltip() {
 	 * @return string a string of html for the tooltip
 	 */
 	public function get_total_order_tooltip() {
+
+		$message = esc_html__( 'Total Ordered counts tickets from orders of any status, including pending and refunded.', 'event-tickets' );
+
+		return tribe( 'tooltip.view' )->render_tooltip( $message );
+	}
+
+	/**
+	 * Get Pending Tooltip per Ticket
+	 *
+	 * @since TBD
+	 *
+	 * @param $ticket_sold object an object of the ticket to get counts
+	 *
+	 * @return string a string of html for the tooltip
+	 */
+	public function get_pending_by_ticket_tooltip( $ticket_sold ) {
+
+		$incomplete_statuses = (array) tribe( 'tickets.status' )->get_statuses_by_action( 'count_incomplete', $ticket_sold['ticket']->provider_class, null, true );
+		ob_start();
```

Comment:
@jesseeproductions Let's get this moved to an admin view too, per @bordoni's feedback